src/databricks/labs/ucx/hive_metastore/table_migrate.py

```diff
@@ -275,7 +299,7 @@ def _migrate_table_create_ctas(self, src_table: Table, rule: Rule, mounts: list[
             return False
         return self._migrate_grants.apply(src_table, rule.as_uc_table_key)
 
-    def _migrate_table_in_mount(self, src_table: Table, rule: Rule):
+    def _migrate_table_in_mount(self, src_table: Table, rule: Rule, managed_table_external_storage: str | None = None):
```

Comment:
changes to this method are irrelevant. table-in-mount is a hack at the moment :) and they're all external, to be honest